web_refresher/static/src/js/pager.esm.js

```diff
@@ -0,0 +1,10 @@
+/** @odoo-module **/
+/* Copyright 2022 Tecnativa - Carlos Roca
+ * License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). */
+import {Pager} from "@web/core/pager/pager";
+import {Refresher} from "./refresher.esm";
+
+// Patch control panel to initialize refresher component
```

Comment:
This comment can go away I think :slightly_smiling_face: Or at least rephrase it with something like: we add the refresher component to the Pager, that is hooked into every control panel.